api/converter_utils.go

```diff
@@ -588,6 +589,10 @@ func (si *ServerImplementation) transactionParamsToTransactionFilter(params gene
 	filter.ApplicationID = uintOrDefault(params.ApplicationId)
 	filter.Limit = min(uintOrDefaultValue(params.Limit, si.opts.DefaultTransactionsLimit), si.opts.MaxTransactionsLimit)
 	filter.Round = params.Round
+	// Integer > math.Int64
+	if filter.AssetID > math.MaxInt64 || filter.ApplicationID > math.MaxInt64 || (filter.Round != nil && uint64(*filter.Round) > math.MaxInt64) {
```

Comment:
I see that there are a lot more fields in a filter which are `uint64` https://github.com/algorand/indexer/blob/develop/idb/idb.go#L196 

Why are these the only ones we need to do checks for as opposed to checking all of the supplied filters?

Reply:
I moved these checks to validateTransactionFilter and added checks for all uint64 fields except for AssetAmount which is type numeric(20) in the DB table.